Automattic/wp-calypso
# Revert "Change DSP widget JS URL (#79762)"

This reverts commit 8bbea99366175b3cd8f4d2b5c069f5374fcf6264.

We have a bug in the Android Jetpack app while loading the widget in the new URL, we are temporary reverting the change until we fix that bug.

## Proposed Changes

* Change `dsp_widget_js_src` back to `https://widgets.wp.com/promote/widget.js`

## Testing Instructions

* Open the Live preview and navigate to Tools->Advertising
* Click on the Promote button
* Verify that the Create Campaign widget loads correctly

## Pre-merge Checklist
